src/types.tsx

```diff
@@ -72,6 +72,10 @@ export type HeaderSubviewTypes =
   | 'center'
   | 'searchBar';
 
+export type HeaderChangeEventType = {
```

Comment:
I would rename this type as well.

Reply:
Renamed in https://github.com/software-mansion/react-native-screens/pull/1802/commits/3c9c0fbe13cb07071e71254d249b2f90a964c3a0 👍